[SIEMINT-77] DDS: Trend Micro Email Security Integration (#17963)
DataDog/integrations-core: trend_micro_email_security/README.md

## Overview

> [Trend Micro Email Security][4] is a cloud-based solution that stops more phishing, ransomware, and Business Email Compromise (BEC) attacks. Solution uses an optimum blend of cross-generational threat techniques, like machine learning, sandbox analysis, data loss prevention (DLP), and other methods to stop all types of email threats.

Reviewer: ```suggestion
[Trend Micro Email Security][4] is a cloud-based solution that stops phishing, ransomware, and Business Email Compromise (BEC) attacks. This solution uses an optimum blend of cross-generational threat techniques, like machine learning, sandbox analysis, data loss prevention (DLP), and other methods to stop all types of email threats.
```

This integration ingests the following logs:

- Policy Events/Detection
- Mail Tracking
- URL Click Tracking
- Audit logs

> Visualize detailed insights into Email traffic analysis, URL click analysis and real time threat detection, security detection and observation, compliance monitoring through the out-of-the-box dashboards.

Reviewer: ```suggestion
Using out-of-the-box dashboards, you can visualize detailed insights into email traffic analysis, URL click analysis, real time threat detection, security detection and observation, and compliance monitoring.
```

## Setup

### Installation

To install the Trend Micro Email Security integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][8] documentation.

**Note**: This step is not necessary for Agent version >= 7.52.0.

Linux command:

```shell
sudo -u dd-agent -- datadog-agent integration install datadog-trend_micro_email_security==1.0.0
```

### Configuration

#### Log collection

**Trend Micro Email Security:**

1. Collecting logs is disabled by default in the Datadog Agent. Enable it in the `datadog.yaml` file:

   ```yaml
   logs_enabled: true
   ```

2. Add this configuration block to your `trend_micro_email_security.d/conf.yaml` file to start collecting your Trend Micro Email Security logs.

   See the [sample trend_micro_email_security.d/conf.yaml][9] for available configuration options.

   ```yaml
   logs:
     - type: tcp
       port: <PORT>
       service: trend-micro-email-security
       source: trend-micro-email-security
   ```

3. [Restart the Agent][1].

4. Configure Trend Micro Email Security to send data to Datadog:
   1. Login into Trend Micro Email Security administrator console.

>    2. Configure syslog server's firewall to accept connections from the IP addresses or CIDR blocks listed in [link][5], allowing Trend Micro Email Security to forward syslog messages.

Reviewer: ```suggestion
   2. Configure syslog server's firewall to accept connections from the IP addresses or CIDR blocks listed in [Configuring Syslog Settings][5], allowing Trend Micro Email Security to forward syslog messages.
```

   3. Follow the [Syslog Server Profiles][6] steps.

>       1. In Steps 4 of the `Syslog Server Profiles` instructions, please use `CEF` format for the `syslog messages`.

Reviewer: ```suggestion
      1. In step 4 of the `Syslog Server Profiles` instructions, please use the `CEF` format for the `syslog messages`.
```

>    4. Then, Follow the [Syslog Forwarding][7] steps.

Reviewer: ```suggestion
   4. Follow the [Syslog Forwarding][7] steps.
```

>       1. In Steps 2 to 5 of the `Syslog Forwarding` instructions, select the syslog server created in above Step iii.

Reviewer: ```suggestion
      1. In steps 2 to 5 of the `Syslog Forwarding` instructions, select the syslog server created in above step iii.
```
